<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1268</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icone.ico</normaloff>../icone.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: rgb(221, 61, 33); /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: rgb(85, 170, 255)
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}

/* make use of negative margins for overlapping tabs */
QTabBar::tab:selected {
    /* expand/overlap to the left and right by 4px */
    margin-left: -4px;
    margin-right: -4px;
}

QTabBar::tab:first:selected {
    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */
}

QTabBar::tab:last:selected {
    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */
}

QTabBar::tab:only-one {
    margin: 0; /* if there is only one tab, we don't want overlapping margins */
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_source">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string notr="true">Fonte</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QScrollArea" name="scrollArea1">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents1">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>1022</width>
               <height>486</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>ClosedHandCursor</cursorShape>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QGraphicsView" name="sourceImageViewer">
                <property name="cursor" stdset="0">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustToContents</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>√Årea de Trabalho</string>
            </property>
            <widget class="QListWidget" name="inputLayerList">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>30</y>
               <width>191</width>
               <height>181</height>
              </rect>
             </property>
             <property name="frameShape">
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
            </widget>
            <widget class="QPushButton" name="actionDownLayer">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>220</y>
               <width>44</width>
               <height>27</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
background-color:rgb(85,170,255);
    min-width: 40px;
}

QPushButton:pressed {
    background-color: rgb(85, 170, 255);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
             </property>
             <property name="text">
              <string>Descer</string>
             </property>
            </widget>
            <widget class="QPushButton" name="actionAddLayer">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>220</y>
               <width>44</width>
               <height>27</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
background-color:rgb(85,170,255);
    min-width: 40px;
}

QPushButton:pressed {
    background-color: rgb(85, 170, 255);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
             <property name="shortcut">
              <string>Return</string>
             </property>
            </widget>
            <widget class="QPushButton" name="actionUpLayer">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>220</y>
               <width>44</width>
               <height>27</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
background-color:rgb(85,170,255);
    min-width: 40px;
}

QPushButton:pressed {
    background-color: rgb(85, 170, 255)
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
             </property>
             <property name="text">
              <string>Subir</string>
             </property>
            </widget>
            <widget class="QPushButton" name="actionDelLayer">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>220</y>
               <width>44</width>
               <height>27</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
background-color:rgb(85,170,255);
    min-width: 40px;
}

QPushButton:pressed {
    background-color: rgb(85, 170, 255);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
             <property name="shortcut">
              <string>Backspace</string>
             </property>
            </widget>
            <widget class="QGroupBox" name="groupBox1">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>270</y>
               <width>191</width>
               <height>221</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Filtros</string>
             </property>
             <widget class="QPushButton" name="actionPlayFilter">
              <property name="geometry">
               <rect>
                <x>120</x>
                <y>20</y>
                <width>71</width>
                <height>31</height>
               </rect>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
background-color:rgb(85,170,255);
    min-width: 40px;
}

QPushButton:pressed {
    background-color: rgb(85, 170, 255);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
              </property>
              <property name="text">
               <string>Aplicar</string>
              </property>
             </widget>
             <widget class="QComboBox" name="inputFilter">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>20</y>
                <width>121</width>
                <height>31</height>
               </rect>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: rgb(85, 170, 255);
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

</string>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QSlider" name="zoom">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>90</y>
                <width>160</width>
                <height>22</height>
               </rect>
              </property>
              <property name="cursor">
               <cursorShape>ClosedHandCursor</cursorShape>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="sliderPosition">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_filters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <kerning>true</kerning>
          </font>
         </property>
         <attribute name="title">
          <string>Filtros</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QWidget" name="filterWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QWidget" name="contextualFilterWidget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <widget class="QGraphicsView" name="previa">
                <property name="geometry">
                 <rect>
                  <x>740</x>
                  <y>100</y>
                  <width>481</width>
                  <height>381</height>
                 </rect>
                </property>
                <property name="cursor" stdset="0">
                 <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">QGraphicsView{
background: rgb(240, 240, 240)

}</string>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustToContents</enum>
                </property>
                <property name="backgroundBrush">
                 <brush brushstyle="NoBrush">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
                <property name="foregroundBrush">
                 <brush brushstyle="NoBrush">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
                <property name="dragMode">
                 <enum>QGraphicsView::NoDrag</enum>
                </property>
                <property name="resizeAnchor">
                 <enum>QGraphicsView::AnchorViewCenter</enum>
                </property>
                <property name="viewportUpdateMode">
                 <enum>QGraphicsView::SmartViewportUpdate</enum>
                </property>
               </widget>
               <widget class="QPushButton" name="pushButton">
                <property name="geometry">
                 <rect>
                  <x>940</x>
                  <y>70</y>
                  <width>82</width>
                  <height>17</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <stylestrategy>PreferDefault</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Pr√©via</string>
                </property>
                <property name="shortcut">
                 <string>Return</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="autoRepeat">
                 <bool>false</bool>
                </property>
                <property name="autoExclusive">
                 <bool>false</bool>
                </property>
                <property name="autoDefault">
                 <bool>true</bool>
                </property>
                <property name="default">
                 <bool>false</bool>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="actionConfirmApplyFilter">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
background-color:rgb(85,170,255);
    min-width: 40px;
}

QPushButton:pressed {
    background-color: rgb(85, 170, 255);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
               </property>
               <property name="text">
                <string>Confirmar?</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_target">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>Sa√≠da</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QScrollArea" name="scrollArea2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>102</width>
               <height>120</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QPushButton" name="actionCopyToSource">
                <property name="minimumSize">
                 <size>
                  <width>84</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
background-color:rgb(85,170,255);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: rgb(85, 170, 255);
}

QPushButton:flat {
    border: none; /* no border for a flat push button */
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}
</string>
                </property>
                <property name="text">
                 <string>Copiar para √Årea de Trabalho</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGraphicsView" name="targetImageViewer"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1268</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
